python/friesian/example/multi_task/run_multi_task.py

```diff
@@ -0,0 +1,234 @@
+        print(k, v)
+
+
+def _parse_args():
+    parser = ArgumentParser(description="Set parameters for multi task demo")
+
+    parser.add_argument('--model_type', type=str, default="mmoe",
+                        help='The multi task model, mmoe or ple.')
+    parser.add_argument('--train_data_path', type=str,
+                        default='path/to/training/dataset',
+                        help='The path for training dataset.')
+    parser.add_argument('--test_data_path', type=str,
+                        default='path/to/testing/dataset',
+                        help='The path for testing dataset.')
+    parser.add_argument('--model_save_path', type=str,
+                        default='path/to/save/the/trained/model',
+                        help='The path for saving the trained model.')
+    parser.add_argument('--cluster_mode', type=str, default="local",
+                        help='The cluster mode, such as local, yarn, standalone or spark-submit.')
+    parser.add_argument('--master', type=str, default=None,
+                        help='The master url, only used when cluster mode is standalone.')
+    parser.add_argument('--executor_cores', type=int, default=8,
+                        help='The executor core number.')
+    parser.add_argument('--executor_memory', type=str, default="12g",
+                        help='The executor memory.')
+    parser.add_argument('--num_executors', type=int, default=4,
+                        help='The number of executors.')
+    parser.add_argument('--driver_cores', type=int, default=2,
+                        help='The driver core number.')
+    parser.add_argument('--driver_memory', type=str, default="8g",
+                        help='The driver memory.')
+    args = parser.parse_args()
+    return args
+
+
+if __name__ == "__main__":
+    args = _parse_args()
+    if args.cluster_mode == "local":  # For local machine
+        sc = init_orca_context(cluster_mode="local",
+                               cores=args.executor_cores, memory=args.executor_memory)
+    elif args.cluster_mode == "standalone":
+        sc = init_orca_context("standalone", master=args.master,
+                               cores=args.executor_cores, num_nodes=args.num_executors,
+                               memory=args.executor_memory,
+                               driver_cores=args.driver_cores,
+                               driver_memory=args.driver_memory)
+    elif args.cluster_mode == "yarn":  # For Hadoop/YARN cluster
+        sc = init_orca_context(cluster_mode="yarn", cores=args.executor_cores,
+                               num_nodes=args.num_executors, memory=args.executor_memory,
+                               driver_cores=args.driver_cores, driver_memory=args.driver_memory,
+                               object_store_memory="80g")
+    elif args.cluster_mode == "spark-submit":
+        sc = init_orca_context("spark-submit")
+    else:
+        ArgumentError(False,
```

Comment:
need to change to invalidInputError, refer to other examples

Reply:
got it